tests/test_dispatcher.py

```diff
@@ -174,6 +174,18 @@ def test_error_handler_that_raises_errors(self, dp):
 
         assert self.count == 1
 
+    def test_default_run_async(self, run_async_bot, dp):
```

Comment:
I don't understand this test …

* according to the new fixture, `run_async_bot.defaults.run_async` is always `True`, so why the `if-else`?
* it doesn't even involve a handler, so it doesn't actually test the behaviour of `Handler.handle_update` recarding `run_async`. This also shows in the `codecov` report: the line marked in red is never executed in the tests

![image](https://user-images.githubusercontent.com/22366557/99266521-21be0600-2823-11eb-90df-beb0a6b224ed.png)

Reply:
Oh i see, ig i should do more study around tests before making it, btw i noticed that `Dispatcher.add_error_handler` also has run_async parameter, i should implement default behaviour in error handler too right?